upgrade/helpers/tasks.py

```diff
@@ -1104,8 +1104,8 @@ def nonfm_upgrade(satellite_upgrade=True,
     # Check what repos are set
     upgrade_type = "satellite" if satellite_upgrade else "capsule"
     run('yum repolist')
-    # Stop katello services, except mongod
-    run('katello-service stop')
+    # Stop foreman-maintain services, except mongod
```

Comment:
```suggestion
    # Stop foreman-maintain services, except mongod
```
is `except mongod` still true ??

Reply:
"katello-service stop" internally call foreman-maintain service, in both cases, mongod service gets to stop.
```
# katello-service stop
Redirecting to 'foreman-maintain service'
Running Stop Services

```